packages/flutter_tools/test/general.shard/android/android_gradle_builder_test.dart

```diff
@@ -716,6 +723,59 @@ void main() {
       AndroidStudio: () => FakeAndroidStudio(),
     });
 
+    testUsingContext('Uses namespace attribute if manifest lacks a package attribute', () async {
```

Comment:
@christopherfujino I'm having some weird results with running this test in VSCode, do you have an idea why this is happening?

Disclaimer: I ran only this test in all cases listed below, so it should not have anything to do with test order and/or sharding.

a) If I put `await AndroidApk.fromProject();` in comment and add an `expect(true, true)` at the end of the test, it passes as expected.
b) If I put `await AndroidApk.fromProject();` in comment and add an `expect(true, false)` at the end, it hangs indefinitely (it should have failed instantly)
c) If I keep `await AndroidApk.fromProject();` but don't have an `expect()` it passes instantly.
d) If I keep `await AndroidApk.fromProject();` and have an `expect(apk?.id, 'com.example.foo')`, it hangs indefinitely.